### PR TITLE
new instructions

### PR DESCRIPTION
**Pull request by narendasan**
new instructions for use with arbor

### CODE REVIEW

**Comment by narendasan** [comment]
Probably need to merge this now to match the arbor pr in groot, can someone do a quick review?

File: README.md [hunk 1]
```diff
@@ -88,23 +88,35 @@ Questions on how to add your app to Groot or use the Groot API:
...
+
+    go install github.com/acm-uiuc/arbor/proxy
     ```
 
     + Install packages
 
     ```sh
-    go install github.com/acm-uiuc/groot/proxy
-
     go install github.com/acm-uiuc/groot/config
```
Comment (bcongdon) [inline README.md]: This should be `groot-api-gateway` now, right?

**Comment by narendasan** [comment]
yeah changed

**Comment by bcongdon** [comment]
👍

**Pull request merged**